src/main/java/hw_3/Task8.java

```diff
@@ -0,0 +1,27 @@
+package hw_3;
+
+public class Task8 {
+        public static void main(String[] args) {
+            String sentence = "Hello world!";
+
+            char firstChar = sentence.charAt(0);
+            System.out.println("First symbol: " + firstChar);
+
+            char lastChar = sentence.charAt(sentence.length() - 1);
+            System.out.println("Last symbol: " + lastChar);
+
+            char[] charArray = sentence.substring(6, 11).toCharArray();
+
+            System.out.print("Array of symbols: ");
+
+            for (char c : charArray) {
+                System.out.print(c + " ");
+            }
+
+            System.out.println();
```

Comment:
Can be removed? If you need to type something from the new line you can use '\n'

Reply:
Done (but i'm not sure if this is correct (and previous one as well))